ngraph/core/src/op/topk.cpp

```diff
@@ -154,6 +155,7 @@ op::v1::TopK::TopK(const Output<Node>& data,
       m_mode{as_enum<Mode>(mode)},
       m_sort{as_enum<SortType>(sort)},
       m_index_element_type{index_element_type} {
+    ov::mark_as_precision_sensitive(input(1));
```

Comment:
can we have a dictionary sensitive operation types and its ports in one place? Now it is hardly distributed and controlled.

Reply:
That was the initial solution but we decided that having such a dictionary in one place is wrong and if the new operation is added it is easier to forget to add it to the dictionary.